src/main/cpp/t02_shoes.cpp

```diff
@@ -28,10 +28,32 @@
 
 #include "t02_shoes.h"
 #include <iostream>
-
+#include <vector>
+#include <algorithm>
 
 using namespace std;
 
 int t02_shoes() {
-    
+    int s, n;
+	cin >> s >> n;
+	vector <int> a(n);
+	for(int i = 0; i < n; i++)
+		cin >> a[i];
+	sort(a.begin(), a.end());
+	int prev_s = 0, count = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'prev_s' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4594423)